Stable-Branch-Strategy.md

```diff
@@ -0,0 +1,95 @@
+Branch and release maintenance for Kata Containers project
+
+# Overview
+
+As detailed in the release documentation, the Kata Containers project makes use of
+semantic versioning.  A release version is described by MAJOR.MINOR.PATCH.  Early in
```

Comment:
This duplicates: https://github.com/kata-containers/documentation/blob/master/Releases.md#versioning.